barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -988,11 +996,30 @@ void avm_prove(const std::filesystem::path& bytecode_path,
  */
 bool avm_verify(const std::filesystem::path& proof_path, const std::filesystem::path& vk_path)
 {
+    using Commitment = AvmFlavorSettings::Commitment;
     std::vector<fr> const proof = many_from_buffer<fr>(read_file(proof_path));
     std::vector<uint8_t> vk_bytes = read_file(vk_path);
-    auto circuit_size = from_buffer<size_t>(vk_bytes, 0);
-    auto num_public_inputs = from_buffer<size_t>(vk_bytes, sizeof(size_t));
-    auto vk = AvmFlavor::VerificationKey(circuit_size, num_public_inputs);
+    std::vector<fr> vk_as_fields = many_from_buffer<fr>(vk_bytes);
+
+    vinfo("initializing crs with size: ", 1);
+    init_bn254_crs(1);
```

Comment:
just curious: why is this initialised with size 1?

Reply:
My understanding of the verification code for Zeromorph is that it only uses the identity point for g1 and a single g2 point to initialize the SRS. I saw that value "1" was also used for "verify_client_ivc".
My thought was to not read unnecessarily data as my assumption is that it would be safe.
Is there any security risk to initialize the CRS with a too small amount of data? Otherwise, I am happy to put it back to the same constant as in the avm_prover.
Thanks for the feedback @maramihali

Reply:
just looked back on what verify client_ivc_verify, makes sense